app/controllers/projects/new.js

```diff
@@ -0,0 +1,119 @@
+import Controller from '@ember/controller';
+import { get, getProperties, observer, set, setProperties } from '@ember/object';
+import { alias, lte } from '@ember/object/computed';
+import { inject as service } from '@ember/service';
+import { isNonValidationError } from 'code-corps-ember/utils/error-utils';
+import recordsList from 'code-corps-ember/utils/records-list';
+
+export default Controller.extend({
+  flashMessages: service(),
+  loadingBar: service(),
+  projectSkillsList: service(),
+  store: service(),
+
+  project: null,
+  showCallout: true,
+
+  isFirstProject: lte('organizationProjectsCount', 1),
```

Comment:
Was wondering if we'd do something like this, but I was also wondering if se should limit the number of projects per organization initially? Like, add an actual field, `organization.projectLimit`, which defaults to 1 initially and we can increase through the admin  UI?

That would mean server-side validations.

Reply:
I'm not too concerned since we still have to approve the projects.